### PR TITLE
Sublanguage support

### PR DESCRIPTION
**Pull request by Yoric**

### CODE REVIEW

**Comment by arai-a** [comment]
looks like the PR has unrelated commits.

**Comment by Yoric** [comment]
Stowaway commits removed!

File: crates/binjs_es6/src/lib.rs [hunk 1]
```diff
@@ -20,11 +20,79 @@ pub mod ast;
 /// Serialization/deserialization utilities.
 pub mod io;
 
+/// A mechanism used to enrich an AST obtained from parsing with additional information
+/// and/or domain-specific rewrites.
+///
+/// See the definition of fields for individual enrichments.
+pub struct Enrich {
+    /// If `true`, introduce `AssertedScope*` information.
+    /// Default: `true`.
```
Comment (arai-a) [inline crates/binjs_es6/src/lib.rs]: can you add some comment why this defaults to `true`, while others are not?

File: src/bin/generate_dictionary.rs [hunk 2]
```diff
@@ -132,6 +139,9 @@ fn main_aux() {
                 .short("o")
                 .takes_value(true)
                 .help("Output directory to use for writing the dictionaries. May be overwritten."),
+            Arg::with_name("show-ast")
+                .long("show-ast")
+                .help("Show the AST before extracting the dictionary from a file"),
```
Comment (arai-a) [inline src/bin/generate_dictionary.rs]: Can you clarify that this is AST of input files?

**Comment by arai-a** [comment]
looks like unrelated commits appear again

**Comment by Yoric** [comment]
Thanks. I have no clue what's going on.

**Pull request merged**